packages/babel-core/src/config/full.ts

```diff
@@ -223,8 +223,8 @@ export default gensync<(inputOpts: unknown) => ResolvedConfig | null>(
   },
 );
 
-function enhanceError<T extends Function>(context, fn: T): T {
-  return function* (arg1, arg2) {
+function enhanceError<T extends Function>(context: any, fn: T): T {
+  return function* (arg1: unknown, arg2: unknown) {
```

Comment:
We can use `Parameters<T>[0]` and `Parameters<T>[1]`

Reply:
It doesn't seem to be, it can't be passed into the Function.

Reply:
It doesn't seem to be, it can't be passed into the Function.

Reply:
you should be able to use Parameters in here if you define proper correct type of `T`


i'm not sure what should be in here, but as simple example you can do something like this:

```ts
function enhanceError<T extends (arg1?: unknown, arg2?: unknown) => Generator<unknown>>(
  context: ConfigContext,
  fn: T,
): T {
  return function* (arg1: Parameters<T>[0], arg2: Parameters<T>[1]) {
```